src/App.tsx

```diff
@@ -1,61 +1,107 @@
+import { useState, ChangeEvent, FormEvent } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [todoTitle, setTodoTitle] = useState<string>('');
+  const [todoUser, setTodoUser] = useState<string>('0');
+  const [todos, setTodos] = useState(todosFromServer);
+  const [userError, setUserError] = useState<boolean>(false);
+  const [titleError, setTitleError] = useState<boolean>(false);
+
+  const title = (event: ChangeEvent<HTMLInputElement>) => {
+    setTodoTitle(event.target.value);
+    setTitleError(false);
+  };
+
+  const person = (event: ChangeEvent<HTMLSelectElement>) => {
+    setTodoUser(event.target.value);
+    setUserError(false);
+  };
+
+  const createTodo = (e: FormEvent) => {
```

Comment:
Please keep using one convention. You were using `event` in two previous handler functions so stick to that